getting_started.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d5682e81-741c-4fbc-bc1f-974a3b93c831",
+   "metadata": {},
+   "source": [
+    "# Getting Started With The Explorer API Client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8049a67b-9477-430b-a9ec-0fea113d7b9e",
+   "metadata": {},
+   "source": [
+    "## Authentication\n",
+    "\n",
+    "The explorer API validates every request it receives by generating a cryptographic digest using the filters and a secret key associated with your account in Altmetric Explorer and comparing it to the value of the `digest` query parameter included with the request.  If the values are the same then a data response will be returned.  If not, then the API will respond with HTTP status `400 Bad Request`.\n",
+    "\n",
+    "This means you need to calculate the digest using the same method as the API server does when you create a request.  The process is quite involved (see the [documentation](https://www.altmetric.com/explorer/documentation/api#authentication) for more details) but the API client handles this part for you as long as you supply your api key and api secret when you create a new `Client` object.\n",
+    "\n",
+    "You can get you api key and secret at https://www.altmetric.com/explorer/settings.\n",
+    "\n",
+    "You can use whatever you like to distribute secrets but, for the purposes of demonstrating how the API works, we are using [python-dotenv](https://pypi.org/project/python-dotenv/) and you must set it up with your own key and secret for the examples to work.\n",
+    "\n",
+    "Simply create a file called `.env` in the same folder as this notebook and add the following lines using your own keys from https://www.altmetric.com/explorer/settings.\n",
+    "\n",
+    "```\n",
+    "API_KEY=xxxxxxxx\n",
+    "API_SECRET=yyyyyyy\n",
+    "```\n",
+    "\n",
+    "Go ahead and do it now and then run the cell below to load your keys into environment variables this notebook can access."
```

Comment:
What does "run the cell" mean? Is that a Jupyter term?

If not we might want to be a bit more specific and say something like "Once you have saved your `.env` file you can run the code snippet below in a python shell (you can start a python shell by typing `pyton` into your terminal)".

Reply:
Yes it's [a Jupyter term](https://nbformat.readthedocs.io/en/latest/format_description.html#cell-types) but we can reword if necessary.  Did you find it hard to understand/figure out though?

Reply:
It was more where do I execute this code, in the console?

Reply:
Ah right.  So does that mean you didn't realise there is a play button in Jupyter that runs the code at first?

Reply:
Ah right, yeah I've not used Jupyter so not sure what it does! But now it makes sense!